en/rocky/8/admin_guide/01-presentation.md

```diff
@@ -0,0 +1,299 @@
+
+<!-- TODO: KDE  screenshot -->
+
+The **KDE** desktop environment is based on the **Qt** component library.
+
+It is traditionally more recommended for users coming from a Windows world.
+
+![Tux - The Linux mascot](images/tux.png)
+
+### Free / Open source
+
+A user of a Microsoft or Mac operating system must purchase a license to use the operating system. This license has a cost, although it is usually transparent (the price of the license is included in the price of the computer).
+
+In the **GNU/Linux** world, the Free Software movement provides mostly free distributions.
+
+**Free** does not mean free!
+
+**Open source**: the source codes are available, so it is possible to consult and modify them under certain conditions.
+
+A free software is necessarily Open Source but the opposite is not true since an Open Source software is amputated of a freedom proposed by the GPL license.
+
+#### GPL Licence (General Public License)
+
+The **GPL License** guarantees the author of a software its intellectual property, but allows modification, redistribution or resale of software by third parties, provided that the source codes are provided with the software. The GPL is the license that came out of the **GNU** (GNU is Not UNIX) project, which was instrumental in creating Linux.
+
+It implies :
+
+* the freedom to run the program, for any purpose;
+* the freedom to study how the program works and adapt it to your needs
+* the freedom to redistribute copies;
+* the freedom to improve the program and publish your improvements, for the benefit of the whole community.
+
+On the other hand, even products licensed under the GPL can be paid for. This is not the product itself, but the guarantee that a team of developers will continue to work on it to make it evolve and troubleshoot errors, or even provide support to users.
+
+## Areas of use
+
+A Linux distribution excels for :
+
+* **A server**: HTTP, email, groupware, file sharing, etc.
+* **Security**: Gateway, firewall, router, proxy, etc.
+* **Central computer**: Banks, insurance, industry, etc.
+* **Embedded system**: Routers, Internet boxes, SmartTV, etc.
+
+Linux is a suitable choice for hosting databases or websites, or as a mail server, DNS, firewall. In short, Linux can do just about anything, which explains the quantity of specific distributions.
+
+## Shell
+
+### Generalities
+
+The **shell**, as known as _command interface_, allows users to send commands to the operating system. It is less visible today, since the implementation of graphical interfaces, but remains a privileged means on Linux systems which do not all have graphical interfaces and whose services do not always have a setting interface.
+
+It offers a real programming language including the classical structures: loops, alternatives and the common constituents: variables, passing of parameters, sub-programs. It allows the creation of scripts to automate certain actions (backups, creation of users, system monitoring, etc.).
+
```

Comment:
I would be remiss if I did not push for the Oxford comma in this line: "...alternatives, and the common constituents.." It would definitely end up there as part of an editing pass, so it's a good idea to get in the habit of using them. I have trouble with this one myself.

Reply:
I am not sure to understand but I think I miss the comma before the "and".
In french, we DON'T write the comma before an "et" (et = and), as there is a space before and after : for example.
So you will probably see this same mistakes multiple times. (example = exemple so same problem ^^).

Thank you for your eagle eye